bazel/python_dependencies.bzl

```
@@ -0,0 +1,10 @@
+load("@rules_python//python:pip.bzl", "pip_parse")
+load("@python3_11//:defs.bzl", "interpreter")
+
+def nighthawk_python_dependencies():
+    pip_parse(
+        name = "python_pip_deps",
```

```suggestion
        name = "nh_pip_deps",
```

our pipset is named `@base_pip3` which is meaningless externally, so im thinking to rename -> `@envoy_pip3`

---

i think this is worth address - probs `@nh_pip3` would be better

---

Thank you, done in 6349726d8423ceee216164ed7e411c5beae9b377.